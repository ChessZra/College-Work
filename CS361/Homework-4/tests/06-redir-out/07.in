file file1
file fileTwo
file file3 >fileTwo
echo Hello>file1;echo World;echo !>file3
file file1;file fileTwo;file file3
cat -ve file1;cat -ve fileTwo;cat -ve file3
md5sum file1>fileTwo;md5sum fileTwo>file3;md5sum file3>file1
cat file1;cat fileTwo;cat file3
rm file1;rm fileTwo;rm file3
