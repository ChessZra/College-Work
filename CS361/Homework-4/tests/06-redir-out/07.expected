cs361sh> file1: cannot open `file1' (No such file or directory)
cs361sh> fileTwo: cannot open `fileTwo' (No such file or directory)
cs361sh> cs361sh> World
cs361sh> file1: ASCII text
fileTwo: ASCII text
file3: ASCII text
cs361sh> Hello$
file3: cannot open `file3' (No such file or directory)$
!$
cs361sh> cs361sh> 5e7dbf8e4fddbe87ac6ef52d56ffb148  file3
09f7e02f1290be211da707a266f153b3  file1
97bdda6d5a56b48864771c862770153e  fileTwo
cs361sh> cs361sh> 
